server/sql_info_fetcher.go

```diff
@@ -215,7 +217,7 @@ func (sh *sqlInfoFetcher) zipInfoForSQL(w http.ResponseWriter, r *http.Request)
 		case <-timer.C:
 			cancelFunc()
 		}
-		err = <-errChan
+		err = dumpCPUProfile(errChan, &buf, zw)
```

Comment:
Call `zw.Create()` after `catchCPUProfile`, for avoiding data race with line 209.